### PR TITLE
Add Pterodactyl Games

### PR DESCRIPTION
**Pull request by Erani0**
Add Barotrauma
Add Black Mesa
Add Citadel Forged with fire
Add Half Life
Add Fistful of Frags
Add Modiverse
Add No More Room in Hell
Add Open Fortress
Add Stormworks Build and Rescue
Add Sven Co-op

### CODE REVIEW

**Comment by bumbummen99** [comment]
Can you sign the commit with git commit --amend --signoff -S please?

File: src/GameQ/Protocols/Blackmesa.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    protected $name = 'blackmesa';
+
+    /**
+     * Longer string name of this protocol class
+     *
+     * @type string
+     */
+    protected $name_long = "Black Mesa";
+
+    /**
```
Comment (bumbummen99) [inline src/GameQ/Protocols/Blackmesa.php]: Please remove unused code

File: src/GameQ/Protocols/Citadel.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    protected $name = 'citadel';
+
+    /**
+     * Longer string name of this protocol class
+     *
+     * @type string
+     */
+    protected $name_long = "Citadel";
+
+    /**
```
Comment (bumbummen99) [inline src/GameQ/Protocols/Citadel.php]: Please remove unused code

File: tests/Protocols/Barotrauma.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ */
+
+namespace GameQ\Protocols;
+
+/**
+ * Barotrauma Protocol Class
+ *
+ * @package GameQ\Protocols
+ */
+class Barotrauma extends Source
```
Comment (bumbummen99) [inline tests/Protocols/Barotrauma.php]: This does not look like a test

File: tests/Protocols/Blackmesa.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+ */
+
+namespace GameQ\Protocols;
+
+/**
+ * Blackmesa Protocol Class
+ *
+ * @package GameQ\Protocols
+ */
+class Blackmesa extends Source
```
Comment (bumbummen99) [inline tests/Protocols/Blackmesa.php]: This does not look like a test

File: tests/Protocols/Citadel.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+ */
+
+namespace GameQ\Protocols;
+
+/**
+ * Citadel Protocol Class
+ *
+ * @package GameQ\Protocols
+ */
+class Citadel extends Source
```
Comment (bumbummen99) [inline tests/Protocols/Citadel.php]: This does not look like a test

File: tests/Protocols/Fof.php [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Fistful of Frags
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Fof.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Hl1.php [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Half Life
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Hl1.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Modiverse.php [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Modiverse
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Modiverse.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Nmrih.php [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for No More Room in Hell
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Nmrih.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Of.php [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Open Fortress
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Of.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Sco.php [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Sven Co-op
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Sco.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

File: tests/Protocols/Stormworks.php [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+     * Test responses for Stormworks
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```
Comment (bumbummen99) [inline tests/Protocols/Stormworks.php]: Please remove unneccesary code.
Only required if you use hostname in the query config

**Review by bumbummen99 — CHANGES_REQUESTED** [review]
Please check the review comments, some tests also look strange

**Comment by Erani0** [comment]
the testfile from Add No More Room in Hell not correct

**Review by bumbummen99 — APPROVED** [review]
LGTM

**Pull request merged**